file_store/src/file_source.rs

```diff
@@ -65,7 +65,7 @@ mod test {
         // 2022-08-05 15:35:55     240363 cell_heartbeat.1658832527866.gz
         // 2022-08-05 15:36:08    6525274 cell_heartbeat.1658834120042.gz
         //
-        let file_store = FileStore::new(None, "us-east-1", "devnet-poc5g-rewards")
+        let file_store = FileStore::new(None, "us-east-1", "devnet-poc5g-rewards", None, None)
```

Comment:
Perhaps at this point a builder pattern would be better? We have quite a few of arguments that are normally going to be `None`

Reply:
Feels overkill to me at the moment, to my knowledge, we only call `new` in 2 places, this test and the `from_settings` function in file_store.rs.